hs/app/reach/embed/sh/_common/ensureConnectorMode.sh

```diff
@@ -0,0 +1,67 @@
+# Makes sure to set REACH_CONNECTOR_MODE to one of:
```

Comment:
Something I want to do is simplify and normalize all these names so that we can more easily programmatically generate this page. I want `$CONN(-(live|devnet|browser))?` to be the only options with `devnet` being the default and `browser` only being allowed when directly using the stdlib